src/OpenTelemetry.Exporter.Console/OpenTelemetry.Exporter.Console.csproj

```diff
@@ -37,6 +37,7 @@
     <Compile Include="$(RepoRoot)\src\Shared\ExceptionExtensions.cs" Link="Includes\ExceptionExtensions.cs" />
     <Compile Include="$(RepoRoot)\src\Shared\SpanAttributeConstants.cs" Link="Includes\SpanAttributeConstants.cs" />
     <Compile Include="$(RepoRoot)\src\Shared\TagTransformer.cs" Link="Includes\TagTransformer.cs" />
+    <Compile Include="$(RepoRoot)\src\Shared\TagTransformerJsonHelper.cs" Link="Includes\TagTransformerJsonHelper.cs" />
```

Comment:
nit: It seems like these two files will always be needed together. Instead of forcing 2 csproj links maybe do some kind of wildcard? Maybe this would work (didn't try it, but based on some other stuff I have done in the past):

```xml
<Compile Include="$(RepoRoot)\src\Shared\TagTransformer*.cs" Link="Includes\%(Filename).cs" />
```